ui/src/containers/SampleListViewContainer.jsx

```diff
@@ -214,32 +213,12 @@ export default function SampleListViewContainer() {
     }
   }
 
-  async function getSamplesFromSC() {
-    const manualSamples = [];
-    Object.values(sampleList).forEach((sample) => {
-      if (sample.location === 'Manual') {
```

Comment:
note : Manual sample are automatically delete , also redundant
and as getSamplesFromSC now being use in the PR, remove it for now.